src/docfx/restore/RestoreGitMap.cs

```diff
@@ -11,67 +11,77 @@ namespace Microsoft.Docs.Build
 {
     internal class RestoreGitMap : IDisposable
     {
+        private readonly string _docsetPath;
+        private readonly IReadOnlyDictionary<PackageUrl, DependencyGitLock> _dependencyGitLock;
```

Comment:
I don;t think it's internal model, it's the contract between docfx and docs pipeline, we are using it for tracking versioning of dependency, not only for restore.

Reply:
And we still used it a lot during restore